### PR TITLE
Beta.10

### LINKED ISSUE #719: Account modal view
- [ ] ENS Resolution
- [ ] Balance
- [ ] Realtime flag?
- [ ] Manual recent transactions?
- [ ] hook / action to open account modal

### LINKED ISSUE #765: Add supported extension wallets to "View More" section
We have enough data collected now in order to populate "View More" section with extension wallets.

* Map wallet ids to related homepage uri's
* Populate "View More" with extensions
* Figure out how to handle explorer pagination / search in this case as we'll have static data mixed in
* Create `isMobile` flag for extensions that have mobile wallets

### LINKED ISSUE #766: Cleanup desktop explorer data
We have good chunk of data in our explorer api now. Unfortunately a lot of entries have bad data. I'll outline some common issues below, but we need to address this before stable release in order to look more polished and stable. Not sure what the best approach here is, but we prob need to go over this manually in both w3m desktop and mobile mode and see what is missing:

1. Some wallets go to 404 pages
2. Some mobile wallets are missing deeplinks
3. Some projects are no longer existing
4. Some projects have rounded logo images (should be square to look good in our ui)
5. There are projects in desktop category that don't have desktop wallets
6. There are projects in mobile category that don't have mobile wallets

### LINKED ISSUE #773: Coinbase name appears on the Metamask option
**Describe the bug**
I have metmask and coinbase google extensions added to my chrome browser. When the web3modal pops up it shows me an installed option with metamask image but name being Coinbase. I guess thats definitely a bug where these extensions installed together as doing

**SDK Version**
 - Version [e.g. v1.9.8 or 2.0.0-alpha.2]

**To Reproduce**
1. add coinbase google extension and create/import your wallet
2. add metmask extension and create/import your wallet
3. open web3modal to see view as shown in attached image
Screenshot 2022-12-15 at 2 11 29 PM

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome

### PR DESCRIPTION
**Pull request by 0xAsimetriq**
## Changes
- closes #765
- Remove custom logic / views associated to wagmi (non WalletConnect and Injected) connectors in favor of using their own defaults
- closes #773
- closes #766
- close #719 (balances, ens, new balance button)
- added new config option for `tokenImages`
- added `html-standalone` example
- Updated dependencies
- Fix wallet de-duplication inside mobile in app browsers
- `walletConnectProvider` will now fall back to rpc's defined in chain config if our rpc does not support that chain id

## Todo
- [x] Document `tokenImages`
- [x] Link to `html-standalone` example from docs
- [x] Tag and release `beta.10`
- [x] Merge associated docs pr https://github.com/WalletConnect/web3modal/pull/770
- [ ] Update web3modal.com to use latest version

### CODE REVIEW

File: packages/ui/src/partials/w3m-walletconnect-qr/index.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class W3mWalletConnectQr extends LitElement {
     return html`
       <div class="w3m-qr-container">
         ${this.uri
-          ? html`<w3m-qrcode size=${this.overlayEl.offsetWidth} uri=${this.uri}> </w3m-qrcode>`
+          ? html`<w3m-qrcode size="${this.overlayEl.offsetWidth}" uri=${this.uri}></w3m-qrcode>`
```
Comment (crypblizz8) [inline packages/ui/src/partials/w3m-walletconnect-qr/index.ts:49]: Minor/ No Action: What was the difference to this ?

**Comment by crypblizz8** [comment]
Bug: FTM + BSC error during ChangeNetwork process

Perhaps because I have no BSC / FTM ? But should error to 0 BSC/FTM?

image

image

image

Error Message on Console
`POST https://rpc.walletconnect.com/v1/?chainId=eip155:250&projectId=8e6b5ffdcbc9794bf9f4a1952578365b 400 (Bad Request)`

Unable to copy paste in time for the toastError Message:
image

**Review by crypblizz8 — APPROVED** [review]
LGTM after FTM / BSC is resolved. Great job and super excited for RC!

File: packages/ui/src/partials/w3m-walletconnect-qr/index.ts (hunk 1 above)
Comment (0xAsimetriq) [reply packages/ui/src/partials/w3m-walletconnect-qr/index.ts:49]: I think this might just be formatting from lit dependency updates

**Comment by 0xAsimetriq** [comment]
Error is caused by our rpc not supporting these chains, will add support at least to example apps before releasing tomorrow 👍

**Pull request merged**